PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -939,15 +939,32 @@ protected function connection_tune($args)
             $this->frame_max = (int)$v;
         }
 
-        // @see https://www.rabbitmq.com/heartbeats.html
```

Comment:
please keep this comment

Reply:
I added comments in `AMQPConnectionConfig` on each constant that explain how it works, so this comment felt like it became redundant. if you prefer to keep it shall I then move it to inside the `case AMQPConnectionConfig::HEARTBEAT_TUNE_LEGACY:` ? and do you then also want a comment in the other `case` statements?

Reply:
also note that the comment contradictory:

```
If either value is 0 (see below), the greater value of the two is used
```

that is the Java client behavior, but the last line states:

```
For BC, this library opts for disabled heartbeat if client value is 0.
```

which is the actual implementation, so it only uses the greater value of the two if the broker value is 0

Reply:
@ramunasd how would you like me to proceed here?